udp/client/clientconn.go

```diff
@@ -111,8 +117,7 @@ func (cc *ClientConn) do(req *pool.Message) (*pool.Message, error) {
 		return nil, fmt.Errorf("invalid token")
 	}
 
-	req.SetMessageID(cc.getMID())
-	req.SetType(udpMessage.Confirmable)
```

Comment:
These lines was mainly used for first request (GET/POST/...) and blockwise transfer. So if you remove them blockwise transfer sends NonConfirmable messages. So I want to suggest to move removing lines to next PR #191 and solve the issue there.

Reply:
Makes sense. I leave it to #191 where CON messages are getting the default for new messages.